### PR TITLE
use central cuda caps

### PR DESCRIPTION
**Pull request by cdeepali**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

requires open-ce/open-ce#499
part of open-ce/open-ce#118

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,14 +25,8 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake {{ cmake }}
     - make
-    # Use pins to control cos6/cos7 match
```
Comment (jayfurmanek) [inline recipe/meta.yaml:28]: These changes should probably be in another PR.
We are ready to merge the hw cap updates, but not these changes.

Comment (cdeepali) [reply recipe/meta.yaml]: Sure, reverted that commit.

Comment (jayfurmanek) [reply recipe/meta.yaml]: ok thanks!

**Review by jayfurmanek — APPROVED** [review]

**Comment by cdeepali** [comment]
@jayfurmanek, do you think this can be merged?

**Comment by jayfurmanek** [comment]
If magma can tolerate comma delimited capabilities instead of spaces, then yes.
